### PR TITLE
updated example, Added disarm and kill

### PR DESCRIPTION
**Pull request by unipheas**

### CODE REVIEW

**Comment by unipheas** [comment]
I've updated the interface a bit. It'll allow a user to arm, takeoff, disarm, and kill the drone. I'm going to work on getting all of the other action protos setup and start integrating RxJS, etc.

**Comment by julianoes** [comment]
Thanks @unipheas, I didn't see this.

**Comment by JonasVautherin** [comment]
Oh, I'm sorry I missed it as well. I'll try it ASAP.

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 node_modules/
 *_pb.js
 lib/
+package-lock.json
```
Comment (julianoes) [inline .gitignore:4]: Why did you add this?
I thought this should be added to git:
https://stackoverflow.com/a/44210813/8548472

Comment (unipheas) [reply .gitignore:4]: That must of been an accident. I don't remember how it was added.

Comment (JonasVautherin) [reply .gitignore:4]: Should we remove it in this PR, or should we merge first and fix that later?

Comment (unipheas) [reply .gitignore:4]: I’ll update this PR today then we can merge.

Comment (unipheas) [reply .gitignore:4]: Actually, I have something weird going on so I'm going to merge this and I'll fix it with a new PR
